packages/app/src/cli/models/extensions/specifications/transform/app_config_webhook.ts

```diff
@@ -24,12 +24,96 @@ export function transformToWebhookConfig(content: object) {
   const serverWebhooks = getPathValue(content, 'subscriptions') as NormalizedWebhookSubscription[]
   if (!serverWebhooks) return webhooks
 
-  const webhooksSubscriptions: WebhooksConfig['subscriptions'] = []
+  const subscriptions = serverWebhooks.map((subscription) => {
+    const {topic, ...otherFields} = subscription
+    return {...otherFields, topics: [subscription.topic]} as WebhookSubscription
+  })
```

Comment:
is the `topic` destructure but using `subscription.topic` and is the spread ordering important?

```suggestion
  const subscriptions = serverWebhooks.map(({topic, ...otherFields}) => {
    return {...otherFields, topics: [topic]} as WebhookSubscription
  })
```